src/controllers/trip.controller.js

```diff
@@ -221,3 +222,52 @@ export const approveOrRejectTripRequest = async (req, res) => {
     );
   }
 };
+export const tripStatistics = async (req, res) => {
```

Comment:
This file is going to be so long, better to import your functions into it if needed.